<?xml version="1.0"?>
<launch>
  <arg name="frame_id" default="scala" />
  <arg name="ip_address" default="10.152.42.111" />
  <arg name="port" default="12004" />
  <arg name="is_fusion" default="false" />
  <arg name="publish_raw_data" default="false" />

  <node pkg="ibeo_scala" type="ibeo_scala" name="ibeo_scala">
    <param name="sensor_frame_id" value="$(arg frame_id)" />
    <param name="ip_address" value="$(arg ip_address)" />
    <param name="port" value="$(arg port)" />
    <param name="is_fusion" value="$(arg is_fusion)" />
    <param name="publish_raw_data" value="$(arg publish_raw_data)" />
  </node>
</launch>
